airqo-platform/AirQo-frontend
# issue-#71-add missing graph file

**What does this PR do?**
This PR fixes issue related to missing graph.js file in master branch. For detail: read issue -  https://github.com/airqo-platform/AirQo-frontend/issues/71 
**What JIRA task is related to this PR?**
NONE
**How do I test out this PR?**
Git clone this repo `cd AirQo-frontend/analytics`. Then`run `npm install` to get all the necessary dependencies. Afterwards, run `npm run local` to test out the system. You should be able to reach the landing page without any error message.
